src/powerlaces/boot_cljs_devtools.clj

```diff
@@ -9,9 +9,8 @@
 
 (def ^:private deps '#{binaryage/devtools binaryage/dirac})
 
-(defn- add-preloads! [in-file out-file]
-  (let [preloads ['devtools.preload 'dirac.runtime.preload]
-        spec (-> in-file slurp read-string)]
+(defn- add-preloads! [preloads in-file out-file]
```

Comment:
Can easily become a multimethod I guess, where we dispatch on the preloads type. Surely overkill :smile: feel free to ignore me if you don't have time, I'll merge it anyways.